sympy/core/tests/test_containers.py

```diff
@@ -150,6 +152,12 @@ def test_iterable_is_sequence():
     assert all(iterable(i, exclude=None) for i in not_sympy_iterable)
 
 
+def test_TupleKind():
+    kind = TupleKind(NumberKind, MatrixKind(NumberKind))
+    assert Tuple(1, 2).kind is TupleKind(NumberKind, NumberKind)
+    assert Tuple(1, Matrix([1, 2])).kind is kind
```

Comment:
It's not a big deal but I would have reordered these lines so that we don't have `kind =` on a line which is not used by the line below.